services/client/src/components/PassOptions/index.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+const Options = () => {
+  return (
+    <div>
+      <div id="options">
+        <div id="returningGuest">
+
+              <span id="clicky">
```

Comment:
Looks like we can still run a code formatter over this file to align indentation.